rep-0003.rst

```diff
@@ -180,6 +179,48 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+
+  - Some C++11 will be enabled (TODO(esteve): need to determine which ones)
```

Comment:
When you mentioned which features we were enabling, I assumed this was related to the different levels of compiler support for c++11 on different platforms. Or we could just say we are requiring c++11 and passing `--std=c++11` but don't focus on features available from specific compilers / platforms for this document.